inference-explainers/inference_onboarding_guide.md

```diff
@@ -0,0 +1,376 @@
+# B&A Inference Basic Setup and Testing Guide
+
+**Authors:** <br> [Steve Gan][17], Google Privacy Sandbox <br>
+
+## Introduction
+
+This document describes how to package and deploy Bidding and Auction services ([B&A system
+design][1]) with machine learning (ML) inference capabilities ([B&A inference overview][2]) in cloud
+and local environments. This guide assumes you have administrative access to your cloud provider
+accounts and are familiar with the deployment process for Bidding and Auction servers and using ML
+models for inference.
+
+## Cloud Deployment
+
+The Bidding and Auction services can be deployed on Google Cloud Platform (GCP) or Amazon Web
+Services (AWS). We provide GCP and AWS support for using Bidding and Auction services with inference
+capabilities. Documentation for packaging and deploying Bidding and Auction services is available in
+the [Bidding and Auction services GCP cloud support and deployment guide][3] (GCP) and the [Bidding
+and Auction services AWS cloud support and deployment guide][4] (AWS).
+
+The inference capabilities require additional steps to configure cloud storage and Terraform-based
+services, and are described in this document. While inference for Bidding and Auction services is
+designed in a generic way, inference support is currently limited to the Bidding service.
+
+To deploy and use the inference capabilities, you will:
+
+-   Build and package the Bidding and Auction servers images that _includes the inference
```

Comment:
sorry, I added the _ only to show the changed part, probably not required in the actual text?

Reply:
Done